src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -233,6 +234,65 @@ export class QueryResults {
     q.tokens = [...this.tokens];
     return q;
   }
+
+  modify(action: Actions, columnName: string, value: React.ReactText) {
```

Comment:
While I am 100% on board with putting this logic in a new function, I don't think it should be part of the actual class itself. It should be a standalone function that takes a QueryResults as a parameter.

Reply:
fixed